### PR TITLE
Improve old gen sprites functionality

### PR DESCRIPTION
**Pull request by Joimer**
Always load mini sprites from bw when no animations or bw.
Delete the now redundant option to enable BW sprites.
You just need no animations option load BW sprites for XY.
Make the option text more fitting to the overall style.
Make sure sprites have the proper size.

### CODE REVIEW

**Comment by Joimer** [comment]
btw, we should probably think of adding all sprite data to a single pokedex-mini, it's just a few bytes more of text we'd be loading and would make it easier and working better.

File: js/battledata.js [hunk 1]
```diff
@@ -856,8 +867,9 @@ var Tools = {
 		}
 		// if there is no entry or enough data in pokedex-mini.js or the animations are disabled or past gen, use the proper sprites
 		gen = (gen === 'xy')? 'bw' : gen;
+		// Do this here in case gen was XY but animations were disabled.
+		Tools.loadSpriteData('bw');
```
Comment (Zarel) [inline js/battledata.js]: "In case" scares me. Can't we wrap this in an `if`-statement detecting that case?

**Comment by Layell** [comment]
BW animated isn't redundant, if anything for animated BW sprites.

Can we keep it.

**Comment by Zarel** [comment]
As far as I can tell, there's no need for this change. As @Layell points out, there are people who actually prefer BW sprites to XY models.

**Pull request closed without merging**